core/server/api_container/server/startosis_engine/startosis_interpreter_idempotent_test.go

```diff
@@ -479,6 +479,154 @@ func (suite *StartosisInterpreterIdempotentTestSuite) TestInterpretAndOptimize_U
 	require.True(suite.T(), scheduledInstruction4.IsExecuted()) // this instruction is not affected, i.e. it won't be re-run
 }
 
+func (suite *StartosisInterpreterIdempotentTestSuite) TestStartosisInterpreterIdempotent_SetService() {
```

Comment:
how much does idempotency matter you think? 

we want - idempotency / edits 

is just edits enough? change image sure

Reply:
Discussed internally over slack